sp800-63b/sec5_authenticators.md

```diff
@@ -112,7 +112,7 @@ If the out of band authenticator sends an approval message over the secondary co
 
 Out of band verifiers SHALL generate a random authentication secret with at least 20 bits of entropy using an approved random number generator. They then optionally signal the device containing the subscriber's authenticator to indicate readiness to authenticate.
 
-Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP (or other software-based) service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
+Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP, chat, or other software-based service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
```

Comment:
I'm referring to offerings by [Apple](https://support.apple.com/kb/PH22363?locale=en_US) and [Google](https://support.google.com/voice/answer/6023920) for example that integrate chat services with SMS services so that the OTPs are relayed to e.g. the user's browser in addition to being sent via SMS.

Services like Google Voice clearly fall under the "VoIP" service that are already covered, but chat services like iMessage aren't VoIP, but nevertheless seem to have exactly the same issue that the "VoIP" reference seems to be attempting to address. Similarly, systems like [Project Fi](https://support.google.com/fi/answer/6159953?hl=en) are virtual mobile operators but, and deliver your text messages to Hangouts, which is a chat service that works across any logged-in device.